javascript-primitives-and-variables/main.js

```diff
@@ -0,0 +1,24 @@
+var fullName;
+var isCool;
+var totalPets;
+var unknown;
+var never;
+
+fullName = 'Dylan De Vei';
+isCool = true;
+totalPets = '3';
+unknown = null;
```

Comment:
In JavaScript, you don't need to create a variable's memory space before assigning a value to it, you can assign the value on the same line you declare the variable.